core/ecschema-editing/src/Merging/KindOfQuantityMerger.ts

```diff
@@ -60,6 +60,17 @@ export async function modifyKindOfQuantity(context: SchemaMergeContext, change:
     throw new Error(`Changing the kind of quantity '${itemKey.name}' persistenceUnit is not supported.`);
   }
 }
+/**
+ * Merges a new presentation format into the target kind of quantity
+ * @internal
+*/
+export async function addPresentationFormat(context: SchemaMergeContext, change: KindOfQuantityPresentationFormatDifference) {
+  for (const presentationFormat of change.difference) {
+    const koqKey = new SchemaItemKey(change.itemName, context.targetSchemaKey);
+    const formatString = await updateOverrideFormat(context, presentationFormat);
+    await context.editor.kindOfQuantities.addPresentationFormatString(koqKey, formatString);
```

Comment:
I think, the better approach would be to parse the format string using the KindOfQuantities parseFormatString method (which should be probably moving as a static method to OverrideFormat) and then use the existing editor method addPresentationOverrideFormat instead of adding another addPresentationFormat editor method.